src/IndicatorsDash.jsx

```diff
@@ -33,7 +33,7 @@ class IndicatorsDash extends React.Component {
     const { activeBlock } = this.props;
     this.setState({ activeBlock });
 
-    const areaId = (activeBlock && activeBlock.id) ? activeBlock.id : 'LLA 0970';
+    const areaId = (activeBlock && activeBlock.id) ? activeBlock.id : 'LLA 86';
```

Comment:
Since we are already loading the list of areas from the backend, this conditional should change. Maybe if activeBlock doesn't have an id it should show an error, or redirect to the home, or something like that. We no longer should show a random block by default.

@alegnaaived @danflop What do you think the behaviour should be?

Reply:
For now, according to erika's suggestion, an error message is showed if there is not an active block on the component.